views/calendar/calendar.hbs

```diff
@@ -15,16 +15,15 @@
         <div class="route-calendar">
             <div class="fullcalendar" id="calendar"></div>
         </div>
-
         {{#userHasPermission "CALENDAR_CREATE"}}
-        {{#embed "lib/components/modal-form" class="create-event-modal" action="/calendar/events/" userId=../userId showVideoconferenceOption=../showVideoconferenceOption}}
-            {{#content "fields"}}
-                {{> "calendar/forms/form-create-event" addCourse="true" addTeam="true" collapseIdCourse="2" collapseIdTeam="3"}}
-            {{/content}}
-        {{/embed}}
+            {{#embed "lib/components/modal-form" class="create-event-modal" action="/calendar/events/" userId=../userId showVideoconferenceOption=../showVideoconferenceOption currentTimezone=../currentTimezone currentTimezoneOffset=../currentTimezoneOffset}}
+                {{#content "fields"}}
+                    {{> "calendar/forms/form-create-event" addCourse="true" addTeam="true" collapseIdCourse="2" collapseIdTeam="3"}}
+                {{/content}}
+            {{/embed}}
         {{/userHasPermission}}
 
-        {{#embed "lib/components/modal-form" method="put" action="/calendar/events/" class="edit-event-modal"}}
+        {{#embed "lib/components/modal-form" method="put" action="/calendar/events/" class="edit-event-modal" currentTimezone=../currentTimezone  currentTimezoneOffset=../currentTimezoneOffset}}
```

Comment:
the hbs files should just receive an iso-string or timestamp

Reply:
when displayed time zone information there could be a tooltip which describes that events have to be defined in school time zone (which differs form user time zone)